ui/raidboss/data/05-shb/raid/e9s.js

```diff
@@ -295,17 +327,105 @@ export default {
     {
       id: 'E9S Wide-Angle Phaser Unlimited 2',
       netRegex: NetRegexes.startsUsing({ id: '560[DE]', source: 'Cloud Of Darkness', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '560[DE]', source: 'Wolke Der Dunkelheit', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '560[DE]', source: 'Nuage De Ténèbres', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '560[DE]', source: '暗闇の雲', capture: false }),
+      netRegexCn: NetRegexes.startsUsing({ id: '560[DE]', source: '暗黑之云', capture: false }),
+      netRegexKo: NetRegexes.startsUsing({ id: '560[DE]', source: '어둠의 구름', capture: false }),
       delaySeconds: 8,
       alertText: (data, _, output) => output[data.phaserOutputs.shift()](),
       outputStrings: phaserOutputStrings,
     },
     {
       id: 'E9S Wide-Angle Phaser Unlimited 3',
       netRegex: NetRegexes.startsUsing({ id: '560[DE]', source: 'Cloud Of Darkness', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '560[DE]', source: 'Wolke Der Dunkelheit', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '560[DE]', source: 'Nuage De Ténèbres', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '560[DE]', source: '暗闇の雲', capture: false }),
+      netRegexCn: NetRegexes.startsUsing({ id: '560[DE]', source: '暗黑之云', capture: false }),
+      netRegexKo: NetRegexes.startsUsing({ id: '560[DE]', source: '어둠의 구름', capture: false }),
       delaySeconds: 12,
       alertText: (data, _, output) => output[data.phaserOutputs.shift()](),
       outputStrings: phaserOutputStrings,
     },
+    {
+      id: 'E9S The Second Art Of Darkness Right',
+      netRegex: NetRegexes.startsUsing({ id: '5601', source: 'Cloud Of Darkness', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '5601', source: 'Wolke Der Dunkelheit', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '5601', source: 'Nuage De Ténèbres', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '5601', source: '暗闇の雲', capture: false }),
+      netRegexCn: NetRegexes.startsUsing({ id: '5601', source: '暗黑之云', capture: false }),
+      netRegexKo: NetRegexes.startsUsing({ id: '5601', source: '어둠의 구름', capture: false }),
+      infoText: (data, _, output) => output['startLeft'](),
+      outputStrings: artOfDarknessOutputStrings,
+      run: (data) => {
+        data.artOfDarknessRemaining = 2;
+        data.artOfDarkness = [];
+        delete data.finalArtOfDarkness;
```

Comment:
Technically this can go with the delete on the charges in their respective `run`s if that makes more sense.

Reply:
This is the initialization for the second art? If so, I think doing it here will be more consistent.  I don't mean to try to add protection against code going awry elsewhere, but having "starting thing, ensure consistent state" vs "depend on previous code to have set things up for us" seems better theoretically.